connexta/eve
# PR Clean Up

#### What does this PR do?

Cleaned up the PR box and switched over to using Material Design.  It looks cleaner now and should display better on the deployed monitor.

#### Who is reviewing it?

@vinamartin @Kjames5269 @bankent1 @alchucam @mcalcote

#### How should this be tested?

Make sure you have the necessary environment variables set up (GITHUB_CLIENT_ID and GITHUB_CLIENT_SECRET) and run using "yarn start"

#### Screenshots
<img width="1014" alt="finalPRBOX" src="https://user-images.githubusercontent.com/20527865/61006743-e1da3b00-a31f-11e9-9753-c4ba8d6e6482.PNG">



#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
